src/charm.py

```diff
@@ -1125,8 +1138,12 @@ def _apache_disable_config(self, conf_name):
         self._run_cli(["a2disconf", conf_name])
         self._start_server()
 
-    def _plugin_swift_reconciliation(self):
-        """Reconciliation process for swift object storage (openstack-objectstorage-k8s) plugin."""
+    def _swift_config(self):
+        """Load swift configuration from charm config.
```

Comment:
A bit of description concerning the legacy openstack configuration conversion would help the comprehension of this method

Reply:
Added, thanks!